### PR TITLE
Move 'Got it' popup to appropriate tree entry

### PR DESCRIPTION
**Pull request by rli**
`GotitTooltip#show` takes a component and a point transformer to determine the location to draw the popup. JTree does not expose the underlying JComponent for a given tree entry, however, we can determine the bounds of a node and calculate its offset to the tree.

image

 
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/devToolsTab/DevToolsToolWindow.kt [hunk 1]
```diff
@@ -154,8 +159,17 @@ class DevToolsToolWindow(private val project: Project) : SimpleToolWindowPanel(t
...
 
-    fun makeServiceChildrenVisible() {
-        TreeUtil.expand(tree, 2)
+    fun showGotIt(node: String?, tooltip: GotItTooltip) {
+        TreeUtil.promiseExpand(tree, 2).onSuccess {
+            node ?: return@onSuccess
+            treeModel.invoker.invoke {
+                val path = treeModel.asSequence().firstOrNull { (it.userObject as? AbstractTreeNode<*>)?.value == node }?.path ?: return@invoke
+                runInEdt {
+                    val pointToRight = tree.getPathBounds(TreePath(path))?.let { Point(it.x + it.width, it.y + it.height / 2) } ?: return@runInEdt
```
Comment (manodnyab) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/devToolsTab/DevToolsToolWindow.kt:168]: Would the width of the panel affect the got it tooltip?

**Comment by manodnyab** [comment]
Does it make sense to have a similar one for the Explorer?

File: jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/devToolsTab/DevToolsToolWindow.kt (hunk 1 above)
Comment (rli) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/devToolsTab/DevToolsToolWindow.kt:168]: no the bounding box is determined by the length of the node text

**Comment by rli** [comment]
> Does it make sense to have a similar one for the Explorer?

there is already a tooltip for the explorer

**Review by manodnyab — APPROVED** [review]

**Pull request merged**